### PR TITLE
Fixes #1691 individual scaling

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Review by msevestre — COMMENTED** [review]
Implementation fairly trivial. Also removed the need to pass generic type to add an expression profile since the expression profile already has a molecule instance

File: src/PKSim.Presentation/Presenters/Individuals/ExpressionProfileSelectionPresenter.cs [hunk 1]
```diff
@@ -86,10 +88,11 @@ public class ExpressionProfileSelectionPresenter : AbstractDisposablePresenter<I
 
       private void refreshExpressionProfilesForMolecule()
       {
-         _allExpressionProfilesForMoleculeType = _projectRetriever.Current.All<ExpressionProfile>().Where(canSelect).ToList();
+         _allExpressionProfilesForMoleculeType = _buildingBlockRepository.All<ExpressionProfile>(canSelect);
       }
 
-      private bool canSelect(ExpressionProfile expressionProfile) => expressionProfile.Molecule.IsAnImplementationOf(_moleculeType);
+      private bool canSelect(ExpressionProfile expressionProfile) =>
+         expressionProfile.Molecule.IsAnImplementationOf(_moleculeType) && Equals(expressionProfile.Species, _species);
```
Comment (msevestre) [inline src/PKSim.Presentation/Presenters/Individuals/ExpressionProfileSelectionPresenter.cs:95]: also filtering by species here

File: tests/PKSim.Tests/Core/DomainHelperForSpecs.cs [hunk 2]
```diff
@@ -69,12 +69,22 @@ public static string SystemTemplateDatabasePath()
          return Path.Combine(AppDomain.CurrentDomain.BaseDirectory, PATH_TO_SRC, "Db\\TemplateDB", "PKSimTemplateDBSystem.mdb");
       }
 
-      public static Individual CreateIndividual()
+      public static ExpressionProfile CreateExpressionProfile<TMolecule>(string speciesName = "Species", string moleculeName = "CYP3A4") where TMolecule : IndividualMolecule, new()
```
Comment (msevestre) [inline tests/PKSim.Tests/Core/DomainHelperForSpecs.cs:72]: those helpers are always so...helping :)

**Comment by msevestre** [comment]
Merging. Please review Fairly straight forward stuff

**Comment by Yuri05** [comment]
looks good

**Pull request merged**